Dockerfile

```diff
@@ -0,0 +1,8 @@
+# Use the official Liquibase image as the base
+FROM liquibase/liquibase:4.29.1
+
+# Add the AWS License Service Extension using the Liquibase Package Manager (LPM)
+RUN lpm update && lpm add liquibase-aws-license-service
+
+# Default command to display Liquibase version
+CMD ["liquibase", "--version"]
```

Comment:
The main dockerfile calls the help command. Should we do the same here?

https://github.com/liquibase/docker/blob/main/Dockerfile#L60